build/props/common.props

```diff
@@ -12,33 +12,44 @@
         <LangVersion>10</LangVersion>
         <Authors>.NET Foundation and Contributors</Authors>
         <PackageReleaseNotes>
-            Silk.NET Winter 2022 Update 1
+            Silk.NET Spring 2023 Update 1 (EARLY PREVIEW)
 
             - Added zero-cost extension methods for ComPtrs, making our DirectX bindings easier to use and higher quality than ever before.
-            - Added WebGPU bindings for Emscripten, Dawn, and WGPU.
+            - Added WebGPU bindings for Dawn and WGPU.
             - Added Direct2D bindings. (massive thank you to @curin)
-            - Added an explicit check against calling Reset on a window while within the render loop of said window.
+            - Added DirectComposition bindings. (again, massive preesh to @curin)
+            - Added d3d11on12 bindings.
+            - Added WGL bindings.
             - Added support for OpenAL disconnect extension. (thanks @okaniku)
```

Comment:
missing [al ext](https://github.com/dotnet/Silk.NET/commit/4b3872e3466463efdf7c84c477a6c64dc3cf80b2) and the win-arm64 patches

Reply:
Fixed.